contrib/utilities/indent

```diff
@@ -35,35 +35,36 @@ if test ! -d source -o ! -d include -o ! -d examples ; then
   exit 1
 fi
 
-# Add the location 'setup_astyle.sh' installs astyle to to the local PATH.
-ASTYLE_PATH="$(cd "$(dirname "$0")" && pwd)/programs/astyle/bin"
-export PATH=$ASTYLE_PATH:$PATH
+# Add the location 'download_clang_format' or 'compile_clang_format'
+# installs clang-format to to the local PATH.
+CLANG_FORMAT_PATH="$(cd "$(dirname "$0")" && pwd)/programs/clang-6/bin"
+export PATH=$CLANG_FORMAT_PATH:$PATH
```

Comment:
Please quote, `PATH="$CLANG_FORMAT_PATH:$PATH"`. Otherwise this fails if someone is smart enough to have a directory with whitespace in their path.